### PR TITLE
Added list of all compilable languages in --help

### PR DESCRIPTION
**Pull request by SupaSibs**
I added some entries to the old --to / -t, as it was uncomplete. I also changed the list of --to and --from into alphabetical order because it's super annoying otherwise.

### CODE REVIEW

**Comment by tomByrer** [comment]
Should `Sveltosis` be listed in --from, or is that too alpha?

https://mitosis.builder.io/?outputTab=G4VwpkA%3D&inputTab=M4NwpgNgLmQ%3D&code=DwZwxgTglgDgLgPgFAAIUBsCmcUDsCGAtpigLwoDkAynJgG6YUDcSwA9ONPMqwCZR1kaYFFwwArjgBGo3gC46%2BdOMykA3gWIBfFGyEoAEpnToA9gEIUASRRh8uFBHEPRKAEqZ8YOABoUANRU%2FKlN0KF4%2FUwgUABkoAEdxcPNWNn5BIA%3D

**Comment by SupaSibs** [comment]
It definitely should, but when I tried to type in -f sveltosis it didnt work idk why

**Comment by SupaSibs** [comment]
It definitely should, but when I tried to type in -f sveltosis it didnt work idk why

**Comment by SupaSibs** [comment]
Yeah, still not working. It's a shame though as I like svelte code more than the lite jsx code

**Comment by SupaSibs** [comment]
It seems that Sveltosis isn't even an option in -f (i checked the commands folder for compile and build)

**Comment by tomByrer** [comment]
> It seems that Sveltosis isn't even an option in -f (i checked the commands folder for compile and build)

Yes, I was suggesting for you to add it in ;)

I'm not sure how much help this will be, but here is the original sveltosis repo:
https://github.com/sveltosis/sveltosis/tree/main/examples/my-component-library

**Comment by SupaSibs** [comment]
Oh, ok. I'll try and check the sveltosis parser in parsers folder and see if there's naything i can use.

**Comment by SupaSibs** [comment]
I found out how to get the parser into the CLI, but what **is* the string_ argument in parseSvelte for? It only seems to be used in the mapAst function, but it's only used for one function. Should it be say the JSON, or the JSON parsed to the string, or a value?

**Comment by SupaSibs** [comment]
Thanks, now I understood it. But I have a question, how do I build/test mitosis without NPM? I'm trying to test the new cli and core to see if they work but I don't know how...

**Review by samijaber — APPROVED** [review]

**Comment by samijaber** [comment]
> Thanks, now I understood it. But I have a question, how do I build/test mitosis without NPM? I'm trying to test the new cli and core to see if they work but I don't know how...

You can test by running the `mitosis` command inside any of the example folders within the monorepo. For example: https://github.com/BuilderIO/mitosis/blob/26a4f4ffdaef0f0001038ed5adbb7f0669a832e8/examples/todo/package.json#L4

You can run `npm exec mitosis compile -f` in this sub-folder, and it should work correctly.

File: packages/core/src/parsers/svelte/index.ts [hunk 1]
```diff
@@ -71,3 +71,4 @@ export const parseSvelte = async function (
   const componentName = path.split('/').pop()?.split('.')[0] ?? 'MyComponent';
   return mapAstToMitosisJson(ast, componentName, string_, usesTypescript);
 };
+export {parseSvelte, mapAstToMitosisJson}
```
Comment (samijaber) [inline packages/core/src/parsers/svelte/index.ts:74]: I think this line is incorrect and causing build failures. You are already exporting `parseSvelte`. You just need to add `export` behind the `mapAstToMitosisJson` declaration

**Review by samijaber — CHANGES_REQUESTED** [review]

File: packages/core/src/parsers/svelte/index.ts (hunk 1 above)
Comment (SupaSibs) [reply packages/core/src/parsers/svelte/index.ts:74]: Damn that totally went over my head. I'm going to make a new PR for this as the topic has gone from "fixing -t in CLI" to "adding Sveltosis to cli", so i'll have to close this one.

**Comment by SupaSibs** [comment]
Closed this PR as the topic has changed from fixing -t to adding sveltosis to cli.

**Pull request closed without merging**